operator-framework/src/test/java/io/javaoperatorsdk/operator/ConcurrencyIT.java

```diff
@@ -48,13 +47,13 @@ public void manyResourcesGetCreatedUpdatedAndDeleted() {
             log.info("Creating {} new resources", NUMBER_OF_RESOURCES_CREATED);
             for (int i = 0; i < NUMBER_OF_RESOURCES_CREATED; i++) {
                 TestCustomResource tcr = integrationTest.createTestCustomResource(String.valueOf(i));
-                integrationTest.getCrOperations().inNamespace(TEST_NAMESPACE).create(tcr);
```

Comment:
This files consist undesired changes related to auto refactoring code using IDEA, but it looks good.